dashboard/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "dashboard",
+  "version": "1.0.0",
+  "description": "Shop service for online-store project",
+  "main": "./src/index.ts",
+  "scripts": {
+    "build": "zx ./build.mjs",
+    "start": "ts-node-dev -r dotenv/config src/index.ts",
+    "test": "jest",
+    "test:coverage": "npm run test -- --coverage",
+    "test:watch": "npm test -- --watchAll --no-cache",
+    "postinstall": "zx ./post-install.mjs"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/cookie-session": "^2.0.48",
+    "@types/cors": "^2.8.17",
+    "@types/express": "^4.17.21",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.11.16",
+    "@types/supertest": "^6.0.2",
+    "jest": "^29.7.0",
+    "rimraf": "^5.0.5",
+    "supertest": "^6.3.4",
+    "ts-jest": "^29.1.2",
+    "ts-node-dev": "^2.0.0",
+    "typescript": "^5.3.3"
+  },
+  "dependencies": {
+    "common": "file:../common",
+    "cookie-session": "^2.1.0",
+    "cors": "^2.8.5",
+    "dotenv": "^16.4.1",
+    "express": "^4.18.2",
+    "express-async-errors": "^3.1.1",
+    "express-validator": "^7.0.1"
+  }
+}
```

Comment:
**Issue**: The 'package.json' file specifies dependencies that are not aligned with the latest secure versions, potentially leading to security vulnerabilities. <br> **Fix**: Update all dependencies to their latest versions after testing for compatibility. <br> **Code Suggestion**: 
 ```
 -    "express": "^4.18.2",
 +    "express": "^4.18.3",
 -    "dotenv": "^16.4.1",
 +    "dotenv": "^16.4.2"
 ```